src/components/common/forms/validationSchema.tsx

```diff
@@ -0,0 +1,44 @@
+import * as yup from "yup";
+
+const required = "All fields are required";
+const contactFormRequired = "Both fields are mandatory";
+
+export const signInValidationSchema = yup
+  .object({
+    email: yup
+      .string()
```

Comment:
Did you consider to use `email()` here? Like [this](https://github.com/jquense/yup#getting-started)

Reply:
I agree! Updated!